lib/Rex/Virtualization/LibVirt/create.pm

```diff
@@ -329,11 +329,15 @@ sub _set_storage_defaults {
           function => "0x0",
         };
       }
-      elsif ( $store->{"bus"} eq "ide" && !exists $store->{"address"} ) {
+      elsif (
+        ( $store->{"bus"} =~ /\Aide|scsi\Z/ && !exists $store->{"address"} ) )
```

Comment:
I believe 3 out of 4 Test::Perl::Critic::Progressive complaints are about the missing `/msx` regex modifiers on this line.

Please add this, and double check functionality.

Reply:
added `/msx` and tested.